src/wp-admin/js/inline-edit-tax.js

```diff
@@ -16,8 +32,13 @@ inlineEditTax = {
 			return false;
 		});
 
-		// prepare the edit row
+		/*
+		 * Prepare the edit row.
+		 *
+		 * @param {Object} e The keyup event that has been triggered.
+		 */
 		row.keyup( function( e ) {
```

Comment:
Maybe add some more description to the event binders, for example @listens or @fires: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions